src/test/java/org/broadinstitute/hellbender/engine/dataflow/transforms/RemoveDuplicateReadVariantPairsUnitTest.java

```diff
@@ -66,7 +66,7 @@
                     KV.of(dupes.get(5).getKey().getUUID(), variant5)
             );
 
-            List<KV<GATKRead, Iterable<Variant>>> finalExpected = testData.getKvReadiVariant();
+            List<KV<GATKRead, Iterable<Variant>>> finalExpected = testData.getKvReadiVariantBroken();
```

Comment:
Does this imply that these tests are broken/giving incorrect results? (same question applies below)